resolwe_bio/processes/variant_calling/wgs_preprocess.py

```diff
@@ -41,10 +42,10 @@ class WgsPreprocess(Process):
         "resources": {
             "cores": 4,
             "memory": 32768,
-            "storage": 400,
+            "storage": 800,
         },
     }
-    data_name = '{{ reads|sample_name|default("?") }}'
+    data_name = '{{ reads|sample_name|default("?") if reads else aligned_reads|sample_name|default("?") }}'
```

Comment:
I am not sure how exactly the templating language works but is this not the same as

'{{ reads|aligned_reads|sample_name|default("?")'